java/libphonenumber/pom.xml

```diff
@@ -47,6 +47,15 @@
             <goals>
               <goal>shade</goal>
             </goals>
+            <configuration>
+              <relocations>
+                <relocation>
+                  <pattern>com.google.protobuf</pattern>
+                  <shadedPattern>com.google.i18n.phonenumbers.repackaged.com.google.protobuf
+</shadedPattern>
```

Comment:
If you want to have it on multiple lines, please split like so:
```
...
<pattern>com.google.protobuf</pattern>
<shadedPattern>
  com.google.i18n.phonenumbers.repackaged.com.google.protobuf
</shadedPattern>
...
```

It could also be on a single line, I had just misread it earlier.